paddle/operators/pool_op.cc

```diff
@@ -80,34 +80,31 @@ Pool2dOpMaker::Pool2dOpMaker(framework::OpProto *proto,
             "the number of channels, H and W is the height and "
             "width of feature.");
 
-  AddAttr<std::string>("pooling_type",
-                       "Pooling_type of pooling operator."
+  AddAttr<std::string>("poolingType",
+                       "(string), poolingType of pooling operator."
                        "Str constant equal to 'max' or 'avg'.")
       .InEnum({"max", "avg"});
-
   AddAttr<std::vector<int>>(
       "ksize",
-      "The pooling window size(height, width) of pooling operator."
-      "If global_pooling = true, ksize is ignored and need not be "
+      "(vector ), the pooling window size(height, width) of pooling operator."
+      "If globalPooling = true, ksize is ignored and need not be "
```

Comment:
Remove "and need not be specified".

Ignored means whether a user is providing this value, it's not used.

"need not be specified" means the user should never provide this value, or else it will cause error.

Reply:
Done